web/libs/editor/src/tags/object/RichText/model.js

```diff
@@ -179,6 +179,17 @@ const Model = types
         }
       },
 
+      onDispose() {
+        self.regs.forEach((region) => {
+          try {
```

Comment:
do we expect errors there so we have to catch them?

Reply:
it makes sense @hlomzik, I was trying to make things safer, but seeing it up close there is no reason for this validation, thanks! :)